client/signup/steps/jpo-site-type/select-business-address.jsx

```diff
@@ -43,40 +109,46 @@ class SelectBusinessAddress extends React.Component {
 				<Card>
```

Comment:
Since you removed `handleBusinessInfo` usages, you have to remove `handleBusinessInfo` instantiation in line 97 since it's no longer used.